WinUIGalleryUnitTests/WinUIGalleryUnitTests.csproj

```diff
@@ -0,0 +1,65 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Do we not need a certificate for the unit types project?

Reply:
We're not creating an msix so we don't need the cert.